sources/packages/backend/apps/db-migrations/src/migrations/1712601354111-UpdateProgramYearDropDownList.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class UpdateProgramYearDropDownList1712601354111
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Update-program-year-drop-down-list.sql", "ProgramYear"),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Rollback-program-year-drop-down-list.sql", "ProgramYear"),
```

Comment:
We usually keep the action we are rolling back in the name: "Rollback-update-program-year-drop-down-list.sql"

Reply:
name updated to include update in the rollback